src/main/java/games/strategy/triplea/delegate/UndoablePlacement.java

```diff
@@ -16,12 +16,15 @@
 /**
  * Contains all the data to describe a placement and to undo it.
  */
-class UndoablePlacement extends AbstractUndoableMove {
+public class UndoablePlacement extends AbstractUndoableMove {
   private static final long serialVersionUID = -1493488646587233451L;
   final Territory m_placeTerritory;
   Territory m_producerTerritory;
   PlayerID m_player;
 
+  /**
+   * Public UndoablePlacement constructor.
```

Comment:
Indeed, though the comment is not useful. Worse it's a potential maintenance cost or something a reader might check for accuracy slowing them down.